### PR TITLE
fix(schematics): add lib module to the parent module

### LINKED ISSUE #216: generate lib command should also add the module to app.module.ts
```
ng g lib home      --routing --parent-module=apps/myapp/src/app/app.module.ts
```

this command added `homeRoutes` to `app.module.ts` but  not the `HomeModule` it self.

Expected:
above  command should also import `HomeModule` into `app.module.ts`

### PR DESCRIPTION
**Pull request by cristianmartinez**
fix #216

### CODE REVIEW

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]
This should only happen if the library is not lazy.

File: packages/schematics/src/collection/library/index.ts [hunk 1]
```diff
@@ -193,7 +193,17 @@ function addChildren(options: NormalizedSchema): Rule {
     const importPath = `@${npmScope}/${options.projectDirectory}`;
 
     insert(host, options.parentModule, [
-      insertImport(sourceFile, options.parentModule, constName, importPath),
+      insertImport(
```
[the remark this answers is not in the document]
Comment (cristianmartinez) [reply packages/schematics/src/collection/library/index.ts]: You're right, I will make the change tomorrow, sorry!

[the remark this answers is not in the document]
Comment (cristianmartinez) [reply packages/schematics/src/collection/library/index.ts]: @FrozenPandaz I just pushed the change. Please let me know what do you think about it.

**Comment by vsavkin** [comment]
@FrozenPandaz any update?

**Comment by FrozenPandaz** [comment]
@cristianmartinez 

My apologies. I looked into the code more and noticed that this `Rule` is already conditionally being called on being **not** lazy. Sorry for my mistake, your original submission was actually perfect :). Would you mind reverting back to it? It should be good after that!

```sh
git reset HEAD~1 --hard
git push -f
```

**Comment by FrozenPandaz** [comment]
@cristianmartinez Sorry I'm not really sure what happened. Can you try and remake the PR. Sorry for the delay, did not see this PR since it was closed

**Comment by cristianmartinez** [comment]
No problem @FrozenPandaz, will try to do it soon!

**Pull request closed without merging**